utils/protobuf-schema-utilities/src/main/java/io/apicurio/registry/utils/protobuf/schema/FileDescriptorUtils.java

```diff
@@ -265,32 +256,35 @@ private static DescriptorProto messageElementToDescriptorProto(
             //Fields are optional by default in Proto3.
             String label = fieldLabel != null ? fieldLabel.toString().toLowerCase() : OPTIONAL;
 
+            String fieldType = null;
             ProtoType protoType = field.getType();
             String fieldTypeName = String.valueOf(protoType);
             ProtoType keyType = protoType.getKeyType();
             ProtoType valueType = protoType.getValueType();
             // Map fields are only permitted in messages
             if (protoType.isMap() && keyType != null && valueType != null) {
                 label = "repeated";
-                fieldTypeName = toMapEntry(field.getName());
+                fieldType = "message";
+                String fieldMapEntryName = toMapEntry(field.getName());
+                fieldMapEntryName = fieldMapEntryName.substring(0, 1).toUpperCase() + fieldMapEntryName.substring(1);
+                fieldTypeName = String.format("%s.%s", messageElem.getType(), fieldMapEntryName);
```

Comment:
Is `fieldTypeName` getting used?

Reply:
Yes, it is used in Line #297 when adding the field.